### DISCUSSION TITLE
Welcome to localization-provider-epi Discussions!

### DISCUSSION THREAD

**Discussion by valdisiljuconoks** [valdisiljuconoks/localization-provider-epi#139] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tibormohr** [comment]
Hi,

I'm using this plugin for a long time but now I got performance issues.

The version that I'm using is 6.4.1 and I wonder if there is a way to use pagination for the table view.

Thanks ahead,

TM

**Reply by valdisiljuconoks** [reply]
Hi,

Sorry for delay here. Performance for the old version of the plugin was an issue for a really long time. I'm going to implement paging for the .NET Core version (7.x) and will see - maybe it's easy enough to back-port it to .NET Fx version as well.

**Comment by tibormohr** [comment]
Hi,

That sounds great :) 
Our optimizely version is 11.
<package id="EPiServer.CMS.AspNet" version="11.14.0" targetFramework="net471" />

Do you have a time plan for when we could expect a new 6.x version with pagination?

Best regards,
TM

**Comment by tibormohr** [comment]
Hi,

 I just discussed this with my company and there is a possibility of paying an amount if it could speed up to fix the problem.

Thanks ahead,
TM

**Reply by valdisiljuconoks** [reply]
Hi,

It's better to send in PR than money :) Summer is hard period, but I'll try to focus on this issue to get it out as soon as I can.

**Comment by tibormohr** [comment]
Hi,

I made some changes to handle the query filter and pagination to: 
'https://github.com/valdisiljuconoks/LocalizationProvider.git/' 
'https://github.com/valdisiljuconoks/localization-provider-epi.git/'

But I'm not authorized.

**Reply by valdisiljuconoks** [reply]
usually changes are made via forks. possible for you to move changes in your private fork?

**Reply by tibormohr** [reply]
Thanks,

I made a PR for LiócalizationProvider with pagination and filter functions and PR for localizationprovider-epi with query filter

**Reply by tibormohr** [reply]
Hi,

I saw that it was merged. Do you know when it will be deployed as a nuget package?

**Reply by valdisiljuconoks** [reply]
see my comment in one of the PR. I'm missing one file

**Comment by valdisiljuconoks** [comment]
version 6.5.3 is out for a publishing.. soon to be available.

thx again for contrib! 👍

**Reply by tibormohr** [reply]
Thx, it was my pleasure

Have a nice day
